From cd780d60b41f5b743e7c4518364053a9c266f17d Mon Sep 17 00:00:00 2001
From: Jan Luebbe <jlu@pengutronix.de>
Date: Tue, 28 May 2013 11:04:12 +0200
Subject: [PATCH 02/17] scripts: Makefile: do not use obj-* variables for
 userspace tools

When using obj-?, it erroneously tries to link a built-in.o. Instead
use the extra-? variable.

Signed-off-by: Jan Luebbe <jlu@pengutronix.de>
---
 scripts/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/Makefile b/scripts/Makefile
index 307dc3d..eaf12d9 100644
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -32,7 +32,7 @@ subdir-	+= basic kconfig setupmbr
 quiet_cmd_csingle	= CC      $@
       cmd_csingle	= $(CC) -Wp,-MD,$(depfile) $(CFLAGS) -o $@ $<
 
-obj-$(CONFIG_BAREBOXENV_TARGET)	+= bareboxenv-target
+extra-$(CONFIG_BAREBOXENV_TARGET) += bareboxenv-target
 
 scripts/bareboxenv-target: scripts/bareboxenv.c FORCE
 	$(call if_changed_dep,csingle)
-- 
1.7.10.4

